src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Properties/Resources.resx

```diff
@@ -117,10 +117,238 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AddingResourceGroup" xml:space="preserve">
+    <value>Adding resource group...</value>
+  </data>
+  <data name="CancelResourceGroupDeployment" xml:space="preserve">
+    <value>Are you sure you want to cancel the active deployment of resource group '{0}'?</value>
+  </data>
+  <data name="CancelResourceGroupDeploymentMessage" xml:space="preserve">
+    <value>Cancelling active resource group deployment ...</value>
```

Comment:
nit: no space after "deployment" and "..." or get line 121 in sync with this line